src/main/java/be/seeseemelk/mockbukkit/MockChunkData.java

```diff
@@ -0,0 +1,126 @@
+package be.seeseemelk.mockbukkit;
+
+import be.seeseemelk.mockbukkit.block.data.BlockDataMock;
+import org.bukkit.Material;
+import org.bukkit.World;
+import org.bukkit.block.data.BlockData;
+import org.bukkit.generator.ChunkGenerator;
+import org.bukkit.material.MaterialData;
+import org.jetbrains.annotations.NotNull;
+
+public class MockChunkData implements ChunkGenerator.ChunkData
+{
+
+	private final BlockData[][][] blocks;
+
+	private final int maxHeight;
+	private final int minHeight; // unused but will be useful in 1.17/1.18
```

Comment:
So... why did you choose to place max before min here...? It would make much more sense the other way.

Reply:
i added it after